core/templates/domain/skill/skill-rights.model.ts

```diff
@@ -18,15 +18,15 @@
 
 export interface SkillRightsBackendDict {
   'can_edit_skill_description': boolean,
-  'skill_id': string
+  'skill_id': string | null
```

Comment:
Probably add a comment at declarations about why ` | null` has been added as a type here, this will help reviewers to review PRs quickly.

Reply:
done